account_payment_order/models/account_move_line.py

```diff
@@ -57,7 +57,11 @@ def _get_linked_move_communication(self):
         if self.move_id.reversal_move_id:
             references.extend(
                 [
-                    move.payment_reference or move.ref
+                    (
+                        move.ref
```

Comment:
It's not mandatory to fill the field:

```suggestion
                        move.ref or move.payment_reference
```